specification/containerservice/resource-manager/readme.md

```diff
@@ -37,16 +37,25 @@ openapi-type: arm
 tag: package-preview-2022-10
 ```
 
+### Tag: package-preview-2022-10-only
+
+These settings apply only when `--tag=package-preview-2022-10-only` is specified on the command line.
+
+``` yaml $(tag) == 'package-preview-2022-10-only'
+input-file:
+  - Microsoft.ContainerService/preview/2022-10-02-preview/managedClusters.json
+```
 
 ### Tag: package-preview-2022-10
 
 These settings apply only when `--tag=package-preview-2022-10` is specified on the command line.
 
 ```yaml $(tag) == 'package-preview-2022-10'
 input-file:
-  - Microsoft.ContainerService/preview/2022-10-02-preview/fleets.json
```

Comment:
Is there no fleet 10-02-preview API? If so, this seems fine but just want to make sure we're not deleting files we shouldn't be.

Reply:
Yes, I've confirmed with fleet team members, there's no plan to add 10-02-preview for fleets.